Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -228,20 +241,7 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("privileged", *sc.Privileged, "Privileged containers are not allowed"))
 	}
 
-	if sc.Capabilities != nil && len(sc.Capabilities.Add) > 0 {
-		for _, cap := range sc.Capabilities.Add {
-			found := false
-			for _, allowedCap := range s.scc.AllowedCapabilities {
-				if cap == allowedCap {
```

Comment:
My concern is that we're setting API behavior we'll have to support long-term, based on a single runtime's behavior. Also, we try to avoid normalization of user input in the API... I'd rather make them say what they mean, but if no one else is bothered by it (or cannot envision a time when we'll need to preserve the case of what the user specifies for a container runtime), then toUpper is better